Code/Tools/AWSNativeSDKInit/source/AWSLogSystemInterface.cpp

```diff
@@ -24,6 +25,9 @@ AZ_POP_DISABLE_WARNING
 
 namespace AWSNativeSDKInit
 {
+    AZ_CVAR(int, bg_awsLogLevel, -1, nullptr, AZ::ConsoleFunctorFlags::Null,
```

Comment:
Q1: Is this a candidate for a sys_ level cvar?

Q2: why is this set to -1? 

Q3: Did we consider just defaulting to a mapping of sys_loglevel?

IMHO This should be set to ReadOnly, it should not be settable at runtime IMHO as developers should be in control, and clients should not change the log level.

Reply:
1. Can you define what do you mean by system level cvar, this cvar is useable for both client, server, editor and cmd
2. Set to -1 so it will fall back to default logging setup with AWSNativeSDKInit
3. Don't really follow this question.
Just follow the original implementation, which checks and adjusts log level during runtime as well.

Reply:
As a developer, I can see the usefulness of being able to change the loglevel at runtime when troubleshooting, primarily for the convenience of not having to restart the game or editor in order to get more information about what's going on.  Sometimes it's difficult to get back into a particular situation to repro an issue.  The legacy CVAR system supported a way of indicating if a CVAR should only exist in non-release builds but I'm not sure if AV_CVAR is intended to handle that, or if we should just be using traits for that.

The AzCore logger log levels appear to be the reverse of AWS log levels
```
enum class LogLevel : int8_t { Trace = 1, Debug = 2, Info = 3, Notice = 4, Warn = 5, Error = 6, Fatal = 7 };
```
And AZ::Debug loglevels are
```
 enum LogLevel : int
        {
            Disabled = 0,
            Errors = 1,
            Warnings = 2,
            Info = 3
        };
```

And `sys_loglevel` appears to be unrelated, defaults to '2' which means 'full' and  maybe not used anywhere in the code anymore - looks like it is legacy?

Reply:
In order to hide the cvar for release, we can wrap it with check on AZ_DEBUG_BUILD and AZ_PROFILE_BUILD

We can't use o3de log level, as aws log level refers to https://sdk.amazonaws.com/cpp/api/0.14.3/aws-cpp-sdk-core_2include_2aws_2core_2utils_2logging_2_log_level_8h_source.html

I don't know what is `sys_loglevel`, as in old world it tracks legacy `sys_SetLogLevel` which is default to 0 based on code

Reply:
We use `bg_traceLogLevel` now in Trace.cpp.
`sys_logLevel` is actually the key used for an AZ::EnvironmentVariable and not a CVar: https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Debug/Trace.cpp#L64